addons/account_payment/controllers/payment.py

```diff
@@ -11,35 +11,43 @@
 
 class PaymentPortal(payment_portal.PaymentPortal):
 
-    @route('/invoice/transaction/<int:invoice_id>', type='json', auth='public')
-    def invoice_transaction(self, invoice_id, access_token, **kwargs):
+    @route('/invoice/transaction', type='json', auth='public')
+    def invoice_transaction(self, invoice_ids, access_token, payment_reference=None, **kwargs):
```

Comment:
It's probably better to add another route to treat the flow of paying overdue invoices, IMO.